Dockerfile

```diff
@@ -7,9 +7,7 @@ RUN apt-get update && apt-get install -y ffmpeg cmake swig libavcodec-dev libavf
 RUN ln -s /usr/bin/ffmpeg /usr/local/bin/ffmpeg
 
 # Copy necessary threatexchange folders
-COPY ./threatexchange/tmk/cpp /app/threatexchange/tmk/cpp
 COPY ./threatexchange/pdq/cpp /app/threatexchange/pdq/cpp
```

Comment:
I think this PDQ line can go as well. It won't cause any problems if it stays, of course.